server/polar/subscription/service.py

```diff
@@ -587,6 +588,41 @@ async def send_confirmation_email(
             from_email_addr="noreply@notifications.polar.sh",
         )
 
+    async def send_cancellation_email(
```

Comment:
Nice, but this isn't called anywhere, right?

Maybe we should add in `update_subscription_from_stripe` above:

```py
elif subscription.cancel_at_period_end:
    await self.send_cancellation_email(session, subscription)
```

Reply:
ah, rookie mistake, code updated!